src/runtime_src/core/tools/xbutil2/SubCmdExamine.cpp

```diff
@@ -55,28 +55,30 @@ namespace po = boost::program_options;
 #include "tools/common/ReportMailbox.h"
 #include "tools/common/ReportQspiStatus.h"
 #include "tools/common/ReportCmcStatus.h"
+#include "tools/common/ReportBOStats.h"
 
 // Note: Please insert the reports in the order to be displayed (alphabetical)
   static ReportCollection fullReportCollection = {
   // Common reports
     std::make_shared<ReportAie>(),
     std::make_shared<ReportAieShim>(),
-    std::make_shared<ReportMemory>(),
-    std::make_shared<ReportHost>(),
-    std::make_shared<ReportDynamicRegion>(),
-    std::make_shared<ReportDebugIpStatus>(),
     std::make_shared<ReportAsyncError>(),
+    std::make_shared<ReportBOStats>(),
```

Comment:
Just wondering, why the change of order. Can't figure out the reason.

Reply:
ascending order changes